### ISSUE TITLE
Synthesize feedback from research sessions with OFA (August 2020)

### ISSUE THREAD

**Issue by shubhi-raft** [raft-tech/TANF-app#113]
Description:
Use the interview notes to perform collaborative analysis and synthesis with PO, 18F, and OFA stakeholders.

AC: 
- [x] Draft research synthesis has been shared with PO and team

Tasks: 

Synthesis
- [x] Gather all notes and fill in gaps with video recording
- [x] Delete video recording once notes are complied
- [x] De-identify data 
- [x] Perform top-line synthesis
- [x] Synthesis workshop with PO/stakeholders
- [x] Document synthesis - Scope (research goal(s)), High-level findings, Actionable learnings, Issues that have to be added to ZenHub, Takeaways for future research
- [x] Document 'What we focussed on', 'Key Takeaways', and 'Opportunities for future research' in Wiki.

DoD: 
 - [x] Documentation: Draft research synthesis has been documented and shared with PO and team
 - [x] @cmajel has reviewed and signed off
 - [x] @lfrohlich has reviewed and signed off

**Comment by matthewjdoty** [comment]
@lfrohlich @cmajel the [high-level findings](https://teams.microsoft.com/l/file/26A72603-0075-4888-B0A7-73890B52761B?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FDesign%2FResearch%20August-September%202020%2FOFA%20Interviews%20high%20level%20findings.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:e92913e3e7d443adb823e6497dff1fb3@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287) are in the design folder and are ready for Gov review. This was the task I wanted to break into its own story but we decided I'd just tag y'all instead.

**Comment by cmajel** [comment]
@matthewjdoty I had some time this morning to do a quick review of the top level findings (see attached). Thanks for being flexible while we're figuring out our Teams access - I've emailed you the file with inline comments. 

Summarizing my comments:

- This feels incomplete as a research readout and it's not clear how these top level findings are supported by observations in the interviews. Can you add more detail and support for these themes? Sub bullets or a short write up would be fine. An example:

> Tribes are more likely to have submission errors than states

It would be helpful to note what we heard about this and why they are more likely to have errors. Or, if we did not hear it, note what we need to follow up on. 

- Similarly, there are places where we could add context and background to some of these topics so that someone new to the team can get up to speed quickly.  Sub bullets or a short write up would be fine. An example:
> No feedback mechanisms exist to inform STTs about the status of their submissions

Is it no mechanism or a highly manual and broken mechanism? They do receive transmission reports and communications from OFA. That would be helpful context for a future team mate on what OFA and STTs currently experience. 

- It would be useful to add an overview of how many folks were talked to and the methods used in the sessions.
- I flagged a few typos in the doc

**Comment by matthewjdoty** [comment]
@cmajel Excellent feedback!
 
Thank you so much. And perfect timing too! All of the items you lay out below (and then some) are part of the detailed findings I’m currently working on but it’s helpful to know that these elements are also expected from a high-level standpoint.
 
With this in mind, It seems like we have a couple of options for moving forward.

1. Update the high-level findings with this information
2. Leave high-level findings as they are and lean on the detailed findings for this information
 
From an efficiency standpoint, option two provides the desired content with no additional effort so I’d typically recommend going that way. I understand, however, if the information is desired in both artifacts and am happy to go with option 1 as well.
 
What do you think? Which option would you like me to move forward with? (NOTE: Regardless of which option we choose to go with, We are planning on including this level of detail in all high-level findings going forward).
 
Again, thank you for this feedback! It helps us become better aligned for future sprints.

**Comment by cmajel** [comment]
@matthewjdoty I'd prefer option 1 where we update the high level findings with this information so that we have one document to review. The AC doesn't make mention of a detailed findings vs top line report, so I think it makes sense for us to have one report going forward. 

To clarify, I don't think we need a huge report for this - adding some sub bullets with supporting evidence and context would keep this lightweight but make it more useful. 

ccing @lfrohlich if she has feedback on this too

**Comment by matthewjdoty** [comment]
Moved the issue to "Gov review". The final document can be found [here](https://teams.microsoft.com/l/file/588709DE-FE1F-49E5-8D52-CBC6E25FA9CD?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA-UserResearch%2FShared%20Documents%2FUser%20Research%2FOFA%20Research%20August%202020%2F%23113%20-%20OFA%20Interviews%20Synthesis.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA-UserResearch&serviceName=teams&threadId=19:3bb622256c0b4792976dc5f1f4654fca@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287).

**Comment by cmajel** [comment]
@matthewjdoty looks good 🎉  I added a few comments to the doc.

The only thing left is a wiki recap, and if you write one for me I can post it to the HHS wiki for now. Can you create an issue to determine where research should live in the future / Raft wiki access as a discussion topic? 

With that follow on issue, I'm cool accepting this and having this in review tomorrow.

**Comment by matthewjdoty** [comment]
Thanks, @cmajel! [Here](https://teams.microsoft.com/l/file/8DFB3D29-D36F-4DBE-B293-3044D22387D0?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA-UserResearch%2FShared%20Documents%2FUser%20Research%2FOFA%20Research%20August%202020%2F%23113%20-%20OFA%20Interviews%20Synthesis%20-%20WIKI.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA-UserResearch&serviceName=teams&threadId=19:3bb622256c0b4792976dc5f1f4654fca@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287) is the wiki write up.

**Comment by matthewjdoty** [comment]
Hey @cmajel, I've created issue #270 to capture your request above

**Comment by lfrohlich** [comment]
This looks really great! Thanks to the team for all of the work on this. I made some edits and comments in the research synthesis and the wiki summary.

**Comment by matthewjdoty** [comment]
Thanks, @lfrohlich!  We'll take a look and update as needed

**Comment by matthewjdoty** [comment]
@lfrohlich + @cmajel We've made updates and responded to comments in both docs.

**Comment by cmajel** [comment]
@matthewjdoty [I've added a starter to the wik](https://github.com/HHS/TANF-app/wiki/Research-Activities)i. I'm waiting on clarification from @lfrohlich on if we're good to add the theme overview from the research.

**Comment by lfrohlich** [comment]
I think this is done. Closing out
